tests/e2e/UsingGKE.md

```diff
@@ -0,0 +1,160 @@
+# Running E2E tests on your own kubernets cluster
+
+* [Step 1: Create a kubernetes cluster](#step-1-create-and-setup-a-kubernetes-cluster)
+* [Step 2: Get cluster credentials](#step-2-get-cluster-credentials)
+* [Step 3: Create Clusterrolebinding](#step-3-create-clusterrolebinding)
+* [Step 4: Export test script variables](#step-4-export-test-script-variables)
+* [Step 5: Run](#step-5-run)
+* [Examples](#examples)
+
+## Step 1: Create and setup a kubernetes cluster
+E2E tests require a Kubernetes cluster. You can create one using the Google Container Engine using
+the following command:
+
+```bash
+gcloud container clusters \
+  create ${CLUSTER_NAME} \
+  --zone ${ZONE} \
+  --project ${PROJECT_NAME} \
+  --cluster-version ${CLUSTER_VERSION} \
+  --machine-type ${MACHINE_TYPE} \
+  --num-nodes ${NUM_NODES} \
+  --enable-kubernetes-alpha \
+  --no-enable-legacy-authorization
+ ```
+
+ - `CLUSTER_NAME`: Whatever suits your fancy, 'istio-e2e' is a good choice.
+ - `ZONE`: 'us-central-f' is a good value to use.
```

Comment:
nit: I think us-central-f is not valid.. us-central1-f is a valid one
https://cloud.google.com/compute/docs/regions-zones/